gcp_pilot/drive.py

```diff
@@ -49,7 +50,26 @@ def get(self, permission_id: str, file_id: str, **kwargs):
         return super().get(fileId=file_id, permissionId=permission_id, **kwargs)
 
 
+class Files(DiscoveryMixin, Drive):
+    @property
+    def get_service(self):
+        return self.client.files()
+
+    def create(self, body: dict, **kwargs):
```

Comment:
Instead of making a plain proxy to the API, gcp-pilot should introduce a layer with parameters to make it easier to use/understand.
We don't need to support every single feature, but we should give them a proper interface.

For ["files"](https://developers.google.com/drive/api/reference/rest/v3/files#File), I'd expected to provide a `name` and its `content`, at least.

Reply:
on other hand, I'd like to use `gcp-pilot` bcause it handles credentials issues pretty well.

i will work on a better method interface soon

Reply:
@joaodaher  I dropped some features from the Files class and kept only the `.list` method for now
I plan to dedicate more time in the future to create and neatly test a few creation approach, such as using MediaFileUpload and MediaInMemoryUpload